tests/testthat.R

```diff
@@ -2,3 +2,37 @@ library(testthat)
 library(rb4model)
 
 test_check("rb4model")
+
+test_fit_and_report <- function(){
```

Comment:
- test function should be in tests/testthat folder
- type use_test("fit_and_report")  in your console and it will create the file for you in the correct folder
- testthat.R should not be changed
- refer to tiffany's tutorial for more info! https://r-pkgs.org/whole-game.html#use_testthat

Reply:
Thank you! Really helpful feedback